src/Components/ButtonPanel.js

```diff
@@ -2,36 +2,47 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 import Button from './Button';
+import './ButtonPanel.css';
 
 export default class ButtonPanel extends Component {
   btns = ['AC', '+/-', '%', '÷', '7', '8', '9', 'X', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=']
 
-  callButton(x, y, sIH) {
-    return this.btns.slice(x, y).map(
-      (btn) => <Button key={btn} press={sIH} name={btn} />,
+  callButton(x, y, sIH, fraction) {
+    if (fraction === '1') {
+      return this.btns.slice(x, y).map(
+        (btn) => <Button fr={fraction} key={btn} press={sIH} name={btn} />,
+      );
+    }
+
+    return (
+      <>
+        <Button fr="1" key="0" press={sIH} name="0" />
+        <Button fr="1" key="." press={sIH} name="." />
+        <Button fr="0" key="=" press={sIH} name="=" />
+      </>
```

Comment:
- [ ] ButtonPanel implements `handleClick` function ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png))

```javascript
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
};
```

- [ ] ButtonPanel passes `handleClick` as `clickHandler` props to each `Button` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/passToButton.png))